mne/time_frequency/tfr.py

```diff
@@ -4208,9 +4308,39 @@ def _prep_data_for_plot(
     freqs = freqs[freq_mask]
     # crop data
     data = data[..., freq_mask, :][..., time_mask]
-    # complex amplitude → real power; real-valued data is already power (or ITC)
+    # handle unaggregated multitaper (complex or phase multitaper data)
+    if taper_weights is not None:  # assumes a taper dimension
+        logger.info("Aggregating multitaper estimates before plotting...")
+        if np.iscomplexobj(data):  # complex coefficients → power
+            data = _tfr_from_mt(data, taper_weights)
+        else:  # tapered phase data → weighted phase data
+            data = (data * taper_weights[np.newaxis, :, :, np.newaxis]).mean(axis=1)
+    # handle remaining complex amplitude → real power
     if np.iscomplexobj(data):
         data = (data * data.conj()).real
     if dB:
         data = 10 * np.log10(data)
     return data, times, freqs
+
+
+def _tfr_from_mt(x_mt, weights):
+    """Aggregate complex multitaper coefficients over tapers and convert to power.
+
+    Parameters
+    ----------
+    x_mt : array, shape (n_channels, n_tapers, n_freqs, n_times)
+        The complex-valued multitaper coefficients.
+    weights : array, shape (n_tapers, n_freqs)
+        The weights to use to combine the tapered estimates.
+
+    Returns
+    -------
+    tfr : array, shape (n_channels, n_freqs, n_times)
+        The time-frequency power estimates.
+    """
+    weights = weights[np.newaxis, :, :, np.newaxis]  # add singleton channel & time dims
+    tfr = weights * x_mt
+    tfr *= tfr.conj()
+    tfr = tfr.real.sum(axis=1)
+    tfr *= 2 / (weights * weights.conj()).real.sum(axis=1)
```

Comment:
This aggregation over tapers for the complex spectra follows the same procedure for computing PSDs and how we handle TFR data in MNE-Connectivity.
However, it does differ to how this aggregation is handled elsewhere in the TFR classes (see #13023), so it would be nice to clarify the correct approach.